From: "J. R. Okajima" <hooanon05@yahoo.co.jp>
Date: Sat, 20 Apr 2013 04:11:07 +0900
Subject: [07/83] aufs: version 3.2.x
Origin: http://sourceforge.net/p/aufs/aufs3-standalone/ci/5646be6742e1c0f164a280c541cdec3b77c3bf92

Signed-off-by: J. R. Okajima <hooanon05@yahoo.co.jp>
[bwh: I'm going to cherry-pick fixes as this branch also includes new
 features, so also remove the snapshot date at this point]
---
 include/linux/aufs_type.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/aufs_type.h
+++ b/include/linux/aufs_type.h
@@ -39,7 +39,7 @@
 
 #include <linux/limits.h>
 
-#define AUFS_VERSION	"3.2-20130204"
+#define AUFS_VERSION	"3.2.x"
 
 /* todo? move this to linux-2.6.19/include/magic.h */
 #define AUFS_SUPER_MAGIC	('a' << 24 | 'u' << 16 | 'f' << 8 | 's')
